File: src2/CourseSession.java
public class CourseSession extends CourseSection{
    // Attributes

	private String day;
	private String startTime;
	private String endTime;
	private String classroom;
    // Constructor    
    public CourseSession(Course course, int sectionID, String day, String startTime, String endTime, String classroom, int capacity) {
    	super(sectionID, course, capacity);
    	this.setDay(day);
    	this.setStartTime(startTime);
    	this.setEndTime(endTime);
    	this.setClassroom(classroom);
    }
    

    
    
    // Method to check if the session time is valid
    public boolean isValidSession() {
        // Checks if startDateTime is before endDateTime
    	return true; //startDateTime.isBefore(endDateTime);
    }
	public String getDay() {
		return day;
	}
	public void setDay(String day) {
		this.day = day;
	}
	public String getStartTime() {
		return startTime;
	}
	public void setStartTime(String startTime) {
		this.startTime = startTime;
	}
	public String getEndTime() {
		return endTime;
	}
	public void setEndTime(String endTime) {
		this.endTime = endTime;
	}
	public String getClassroom() {
		return classroom;
	}
	public void setClassroom(String classroom) {
		this.classroom = classroom;
	}
    

}

File: Iteration3/Python/CourseSession.py
class CourseSession(CourseSection):
    def __init__(self, section_id=None, course=None, capacity=None, day=None, start_time=None, end_time=None, classroom=None):
        super().__init__(section_id=section_id, course=course, capacity=capacity)
        self.day = day
        self.start_time = start_time
        self.end_time = end_time
        self.classroom = classroom

    def is_valid_session(self):
        # Placeholder implementation
        return True

    def get_day(self):
        return self.day

    def set_day(self, day):
        self.day = day

    def get_start_time(self):
        return self.start_time

    def set_start_time(self, start_time):
        self.start_time = start_time

    def get_end_time(self):
        return self.end_time

    def set_end_time(self, end_time):
        self.end_time = end_time

    def get_classroom(self):
        return self.classroom

    def set_classroom(self, classroom):
        self.classroom = classroom
